redis/src/cluster.rs

```diff
@@ -509,52 +485,101 @@ where
         Ok(result)
     }
 
+    fn exectue_on_all<'a>(
```

Comment:
```suggestion
    fn execute_on_all<'a>(
```
Same issue w/ following methods.